web/explorer/vite.config.js

```diff
@@ -8,7 +8,7 @@ export default defineConfig(({ command, mode }) => {
     const isBundle = mode === "bundle";
 
     return {
-        base: isBundle ? "/" : "/capa/",
+        base: isBundle ? "/" : "/capa/explorer",
```

Comment:
```suggestion
        base: "./",
```

I think this might do it.
via: https://stackoverflow.com/a/69746868

Reply:
this fix works.

tried it here: https://github.com/williballenthin/capa/blob/wb/webui-actions-1/web/explorer/vite.config.js
which now works here: https://williballenthin.github.io/capa/explorer/#/

Reply:
That works. The single page build `npm run build:bundle` remains the same as well.

Reply:
sweet, thanks for confirming